torch/ao/quantization/quantize_fx.py

```diff
@@ -450,6 +455,7 @@ def prepare_fx(
             "preserved_attributes": ["preserved_attr"],
           }
 
+      * `example_inputs`: (required) Example inputs for forward function of the model
```

Comment:
Since this is required, maybe move it above `prepare_custom_config_dict` to be consistent with the argument order?